flang/lib/Lower/PFTBuilder.cpp

```diff
@@ -23,6 +23,17 @@ static llvm::cl::opt<bool> clDisableStructuredFir(
     "no-structured-fir", llvm::cl::desc("disable generation of structured FIR"),
     llvm::cl::init(false), llvm::cl::Hidden);
 
+// FIXME: should be set with switch such as `--std=2018`.
```

Comment:
Note also that typically enabling OpenMP switches the global default to recursive.  Often compiler have a specific options to compile all routines for possible recursive execution.